### DISCUSSION TITLE
Legend overlapping with network nodes in term visualizations? How to resolve the overlap issues?

### DISCUSSION THREAD

**Discussion by junli1988** [egeulgen/pathfindR#141] [Q&A] [answered]
Hi, this is probably a stupid question. But how do I prevent the legend from overlapping with random network nodes in the term visualization graphs? Is there a command that could enable me to fine-tune the term graphs? In this graph pasted below, the overlap is so bad that I can't even see the gene name.  I'm still quite a noob to RStudio and this package. How did everyone else using this package resolve the issue? As much as I'd love to use the results from this package, this issue with term visualizations has made some of the graphs unable to be presented. 

![image](https://user-images.githubusercontent.com/29934049/186267769-ad085f70-1ea8-4dc5-85cc-4c5ab8a1ffcb.png)

**Comment by egeulgen** [comment, marked as answer]
Hello,

Unfortunately, there is currently no way to prevent this overlap. I'll try and work on this, and update you if I manage to find a solution for this issue

**Reply by junli1988** [reply]
Hi Ege, thank you for the reply. If there's no way to prevent overlap, then is there a way to toggle the legend as "FALSE" so that it doesn't show up? Kind of like what we see in ggplot2 and ComplexHeatmap? I look forward to the updates on the package. Again, thanks for the help.

**Reply by egeulgen** [reply]
Just added the `show_legend` argument to `visualize_term_interactions()`, you can now toggle the legend to `FALSE` by setting this argument in either `visualize_term_interactions()` or `visualize_terms()`.
To use this functionality, update to the latest development version (1.6.3.9008) via `devtools`.

**Comment by junli1988** [comment]
Thanks!
